docs/guides/connect-aepp-to-wallet.md

```diff
@@ -1,90 +1,109 @@
 # Connect an æpp to a wallet
 
-## Introduction
+This guide describes the 4 steps that are necessary to connect your application to a wallet using the RPC API.
 
-This guide describes the 3 steps that are necessary to connect your application to a wallet using the RPC API.
+## Prerequisites
 
-Note:
-
-- The steps below are snippets taken from the full implementation of the [Simple æpp](https://github.com/aeternity/aepp-sdk-js/tree/master/examples/browser/aepp)
+- Install [Superhero Wallet extension](https://wallet.superhero.com/) for simplicity of example.
+You can build your own wallet in the next example
 
-## 1. Initialize the `RpcAepp` Stamp
+## 1. Specify imports and constants and state
 
 ```js
-import { RpcAepp, WalletDetector, BrowserWindowMessageConnection } from '@aeternity/aepp-sdk'
+import { RpcAepp, WalletDetector, BrowserWindowMessageConnection, Node } from '@aeternity/aepp-sdk'
+
+const TESTNET_NODE_URL = 'https://testnet.aeternity.io'
+const MAINNET_NODE_URL = 'https://mainnet.aeternity.io'
+const COMPILER_URL = 'https://compiler.aepps.com';
 
+export default {
+  data: () => ({
+    aeSdk: null,
+    publicAddress: '',
+    balance: 0,
+    node: null,
+    connectedAccounts: null
+  }),
+}
```

Comment:
I am thinking of adding simple template just to print state variables.
There is no visual output of this example now.

Reply:
I think this guide was initially made vue-agnosting, not sure that we need to introduce specific stuff from these, just define a set of functions with comments.